src/Momento.Sdk/MomentoSigner.cs

```diff
@@ -99,11 +99,11 @@ private string CreateJwtToken(JwtPayload jwtPayload)
     }
 
 
-    private JwtPayload CommonJwtBody(string cacheName, string cacheKey, uint expiryEpochSeconds)
+    private JwtPayload CommonJwtBody(string cacheName, string cacheKey, TimeSpan expiryEpoch)
     {
         return new JwtPayload()
             {
-                { "exp", expiryEpochSeconds },
+                { "exp", (long)expiryEpoch.TotalSeconds },
```

Comment:
same comment here about convert vs. cast.

Reply:
Fixed in [d077b1a](https://github.com/momentohq/client-sdk-dotnet/pull/286/commits/d077b1a51c992a9a81dd8dfc0ae1dc0716301ff2)